test/integration/support.js

```diff
@@ -11,7 +11,11 @@ const CLEANUP_TIMEOUT = Number.parseInt(process.env.SEQ_TEST_CLEANUP_TIMEOUT, 10
 
 let runningQueries = new Set();
 
+const _OriginalTz = process.env.TZ;
+
 before(function () {
+  // set code timezone to same as database to avoid TZ errors when comparing dates
+  process.env.TZ = 'UTC';
```

Comment:
This is something I didn't know much about.  So the database is in UTC, but I think my node environment (e.g., `process.env`) was in EST. The difference between timezones broke datetime tests.  I initially placed it closer to the tests, but I thought it might be a better general dev configuration to be at the higher (highest) level.

Reply:
My timezone isn't UTC either and the tests aren't breaking on my end. That's strange.

But I don't see what uses that variable. It's not referenced anywhere else in our codebase and isn't a standard node env variable.

Reply:
The `TZ` is used internally by node, which means anywhere it uses `Date()`, it's going to reference local timezone if set.

https://stackoverflow.com/a/46107140/10408280

Reply:
Thanks, I did not know that :)